portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,6 +45,9 @@
  * Linked to <form> element on index.html with id='comment-form' */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  // Supported upload file types of the comment form
+  private final HashSet<String> FILETYPES = new HashSet<String>(
```

Comment:
Is this better practice than defining the HashSet directly in the getBlobKey function?